server/app/forms/ProgramForm.java

```diff
@@ -90,6 +102,34 @@ public String getLocalizedDisplayDescription() {
     return localizedDisplayDescription;
   }
 
+  public String getLocalizedShortDescription() {
+    return localizedShortDescription;
+  }
+
+  public ImmutableList<String> getAllApplicationStepTitles() {
+    return ImmutableList.of(getApplyStep1Title(), getApplyStep2Title());
+  }
+
+  public ImmutableList<String> getAllApplicationStepDescriptions() {
+    return ImmutableList.of(getApplyStep1Description(), getApplyStep2Description());
+  }
+
+  public String getApplyStep1Title() {
+    return applyStep1Title;
+  }
+
+  public String getApplyStep1Description() {
+    return applyStep1Description;
+  }
+
+  public String getApplyStep2Title() {
+    return applyStep2Title;
+  }
+
+  public String getApplyStep2Description() {
```

Comment:
It's pretty annoying to have to pass all of these titles and descriptions in separately, but I couldn't think of another way to do it. Open to suggestions!